src/Compilers/CSharp/Portable/Parser/Lexer_RawStringLiteral.cs

```diff
@@ -330,8 +331,12 @@ private void ScanMultiLineRawStringLiteral(ref TokenInfo info, int startingQuote
 
             // Skip the leading whitespace that matches the terminator line and add any whitespace past that to the
             // string value.
+#if NETCOREAPP
+            _builder.Append(currentLineWhitespace, indentationWhitespace.Length, Math.Max(0, currentLineWhitespace.Length - indentationWhitespace.Length));
```

Comment:
Feels like we're being tricky here. If the `indentationWhitespace` is longer than `currentLineWhitespace`, we end up with `count` argument of `0` so no copy. So the fact that we passed an invalid `startIndex` in that scenario doesn't matter.
Maybe this would be clearer with an `if` statement (`if (indentationWhitespace.Length < currentLineWhitespace.Length) { _builder.Append(...); }`.

nit: Consider naming arguments for the `Append`.